network-store-iidm-impl/src/main/java/com/powsybl/network/store/iidm/impl/ConnectDisconnectUtil.java

```diff
@@ -50,7 +50,7 @@ static boolean connectAllTerminals(AbstractIdentifiableImpl<?, ?> identifiable,
             if (terminal.isConnected()) {
                 // If the terminal is already connected, log and continue on other terminals
                 reportNode.newReportNode()
-                    .withMessageTemplate("alreadyConnectedTerminal", "A terminal of identifiable ${identifiable} is already connected.")
+                    .withMessageTemplate("core.iidm.network.alreadyConnectedTerminal")
```

Comment:
Why removing second argument ?

Reply:
it is deprecated it will be in a separated file